specification/cognitiveservices/HealthInsights/healthinsights.openapi/service.tsp

```diff
@@ -21,6 +21,7 @@ using TypeSpec.Versioning;
 namespace AzureHealthInsights;
 
 enum ApiVersion {
+  @useDependency(Azure.Core.Versions.v1_0_Preview_1)
```

Comment:
What does this mean?  Is it configuring this spec to use the older linter rules?

Reply:
It is locking the specific api-version to a specific version of Azure.Core types.  Linting rules are not affected.